forge/containers/stub/index.js

```diff
@@ -275,32 +279,125 @@ module.exports = {
         if (!list[instance.id] || list[instance.id].state === 'suspended') {
             throw new Error('Cannot access instance files')
         }
-        return {
-            meta: {},
-            files: [],
-            count: 0
+        if (files[instance.id]) {
+            const pathDots = filePath.replace('/', '.')
+            const response = {
+                meta: {},
+                files: [],
+                count: 0
+            }
+            try {
+                const dir = pathDots ? nrUtil.util.getObjectProperty(files[instance.id], pathDots) : files[instance.id]
+                Object.keys(dir).forEach(entry => {
+                    if (typeof dir[entry] === 'object') {
+                        response.files.push({
+                            name: entry,
+                            type: 'directory',
+                            lastModified: new Date().toISOString()
+                        })
+                    } else {
+                        response.files.push({
+                            name: entry,
+                            type: 'file',
+                            size: dir[entry].length,
+                            lastModified: new Date().toISOString()
+                        })
+                    }
+                    response.count++
+                })
+                return response
+            } catch (err) {
+                if (err.message === 'Cannot convert undefined or null to object') {
+                    const newErr = new Error('not found')
+                    newErr.statusCode = 404
+                    throw newErr
+                } else {
+                    throw err
+                }
+            }
+        } else {
+            return {
+                meta: {},
+                files: [],
+                count: 0
+            }
         }
     },
 
     updateFile: async (instance, filePath, update) => {
         if (!list[instance.id] || list[instance.id].state === 'suspended') {
             throw new Error('Cannot access instance files')
         }
+        if (!files[instance.id]) {
+            files[instance.id] = {}
+        }
+        // const pathDots = filePath.replace('/','.')
+        // const dir = pathDots ? nrUtil.util.getObjectProperty(files[instance.id], pathDots) : files[instance.id]
```

Comment:
```suggestion
```

Dead code

Reply:
That will be needed as more tests get added